.github/workflows/misc-approve-and-merge-pr.yml

```
@@ -0,0 +1,54 @@
+# Merge Update Version PR Workflow
+# This workflow automates the process of merging and approving a pull request for updating the version of a repository.
+
+name: Merge Update Version PR
+
+on:
+  workflow_call:
+    inputs:
+      pull-request-number:
+        required: true
+        type: string
+      repository:
+        required: true
+        type: string
+
+jobs:
+  approve-and-merge-pr:
+    runs-on: ubuntu-latest
+
+    steps:
+      # Displaying the input values for reference
+      - name: Log inputs
+        run: |
+          {
+            echo "# Inputs"
+            echo "| Key                 | Value                             |"
+            echo "| ------------------- | --------------------------------- |"
+            echo "| pull-request-number | ${{ inputs.pull-request-number }} |"
+            echo "| repository          | ${{ inputs.repository }}          |"
+          } >> $GITHUB_STEP_SUMMARY
+
+      # Checking out the specified repository
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          repository: felleslosninger/${{ inputs.repository }}
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      # Approving the pull request using GitHub CLI
+      - name: Auto approve
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN}}
+        run: |
+          gh pr review --approve "${{ inputs.pull-request-number }}"
+          echo "- :white_check_mark: PR ${{ inputs.pull-request-number }} approved" >> $GITHUB_STEP_SUMMARY
+
+      # Merging the pull request, squashing commits, and deleting the branch
+      - name: Pull Request Activate Automerge
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          sleep 3
+          gh pr merge --squash --auto --delete-branch "${{ inputs.pull-request-number }}"
+          echo "- :white_check_mark: PR ${{ inputs.pull-request-number }} merged" >> $GITHUB_STEP_SUMMARY
```

La oss teste dette for å fikse shellcheck

```suggestion
          echo "- :white_check_mark: PR ${{ inputs.pull-request-number }} merged" >> "$GITHUB_STEP_SUMMARY"
```